sdk/python/setup.py

```diff
@@ -26,9 +26,10 @@
     name=NAME,
     version=VERSION,
     description='KubeFlow Metadata SDK',
+    long_description='Please visit the document page: https://github.com/kubeflow/metadata/tree/master/sdk/python',
```

Comment:
since below url points to the github repo, do you think that's better to piont to readthedos url for the sdk?